R/skip.R

```diff
@@ -44,13 +44,19 @@ skip_if_not <- function(condition, message = deparse(substitute(condition))) {
 
 #' @export
 #' @param pkg Name of package to check for
+#' @param minimum_version Minimum required version for the package
 #' @rdname skip
-skip_if_not_installed <- function(pkg) {
-  if (requireNamespace(pkg, quietly = TRUE)) {
-    return(invisible(TRUE))
+skip_if_not_installed <- function(pkg, minimum_version = NULL) {
+  tryCatch(
```

Comment:
Or I could call requireNamespace() in addition to the manual version check -- this would work even if the installed package is somehow damaged so that DESCRIPTION exists but the package cannot be loaded.